### PR TITLE
Add `tvos` as a new platform

### PR DESCRIPTION
**Pull request by neonichu**

### CODE REVIEW

**Comment by neonichu** [comment]
cc @segiddins

**Comment by kylef** [comment]
:+1:

File: lib/xcodeproj/xcodebuild_helper.rb [hunk 1]
```diff
@@ -45,13 +52,16 @@ def parse_sdks_if_needed
         @versions_by_sdk = {}
         @versions_by_sdk[:osx] = []
         @versions_by_sdk[:ios] = []
+        @versions_by_sdk[:tvos] = []
         @versions_by_sdk[:watchos] = []
         if xcodebuild_available?
           sdks = parse_sdks_information(xcodebuild_sdks)
+          p sdks
```
Comment (segiddins) [inline lib/xcodeproj/xcodebuild_helper.rb]: Debug statement

**Comment by segiddins** [comment]
Should we also bump the other versions for the GM?

**Comment by orta** [comment]
Let's do itttttttt

**Comment by segiddins** [comment]
@neonichu 👍🏻 other than my one comment

**Comment by segiddins** [comment]
I've rebased this

**Pull request merged**